.github/workflows/codeql.yml

```diff
@@ -0,0 +1,45 @@
+name: "CodeQL"
+
+on:
+  workflow_call:
+    secrets:
+      CUSTOM_GITHUB_TOKEN:
```

Comment:
Do you know if we need this for the workflow? We could add extra permissions in the job permission instead.

Reply:
Yes, we need it for the `security-events: write`. `CUSTOM_GITHUB_TOKEN` is passed from the caller workflow and will use default `GITHUB_TOKEN`.

Reply:
But you already customized that permission at https://github.com/longhorn/gh-configs/pull/2/files#diff-12783128521e452af0cfac94b99b8d250413c516ec71fe6d97dbea666ff7ba27R29. Do we still need the caller to provide a customized github token?